### PR TITLE
[AIRFLOW-4493]add region_name to emr add step operator

### PR DESCRIPTION
**Pull request by natsumehu**
Make sure you have checked _all_ steps below.

### Jira

- [ ] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW-4493) issues and references them in the PR title. 

### Description

- [ ] We add an extra parameter region_name to the EmrAddStepsOperator.

### Tests

- [ ] My PR quiet simple. We tested on our prod environment and can get the job flow id if our cluster located in no default region.

### Code Quality

- [ ] Passes `flake8`

### CODE REVIEW

File: airflow/contrib/operators/emr_add_steps_operator.py [hunk 1]
```diff
@@ -42,16 +42,18 @@ def __init__(
             self,
             job_flow_id,
             aws_conn_id='s3_default',
+            region_name=None,
```
Comment (feluelle) [inline airflow/contrib/operators/emr_add_steps_operator.py:45]: Please add this new argument to the docs above :)

**Review by feluelle — CHANGES_REQUESTED** [review]
LGTM except the new field is missing in the docs.

(If you don't mind you could also check that the region_name (and all the other args) have been passed to the mocked boto object)

**Comment by ashb** [comment]
This can be specified in side the connection - do we need to add it here too?

**Comment by ashb** [comment]
Region name can be specified in the extra field of the AWS connection. See https://airflow.apache.org/1.10.3/howto/connection/aws.html

**Pull request closed without merging**